src/cdp/connection.ts

```diff
@@ -377,6 +376,8 @@ class CDPSession {
   }
 }
 
+const waitWrapper = makeWaitForNextTask();
```

Comment:
This can be stored as a property on the Connection instance I believe. Avoids extra waiting in case we have multiple active CDP connections (e.g. multiple debug sessions going)

Reply:
👍